### ISSUE TITLE
Observ state seems to be lost after a while

### ISSUE THREAD

**Issue by balu-** [balu-/philips-airpurifier-coapMqtt#1]
After some time the airpurifier seems to not send any more updates.

**Comment by balu-** [comment]
I should take a look at https://github.com/betaboon/philips-airpurifier-coap/pull/57

**Comment by balu-** [comment]
https://datatracker.ietf.org/doc/html/rfc7641#section-3.3.1

```
   To make sure it has a current representation and/or to re-register
   its interest in a resource, a client MAY issue a new GET request with
   the same token as the original at any time.  All options MUST be
   identical to those in the original request except for the set of ETag
   Options.  It is RECOMMENDED that the client does not issue the
   request while it still has a fresh notification/response for the
   resource in its cache.  Additionally, the client SHOULD at least wait
   for a random amount of time between 5 and 15 seconds after Max-Age
   expired to reduce collisions with other clients.
```

**Comment by balu-** [comment]
fixed by #3958c2b711411a908b4b4a87b73338b7355200da 
and changes in underlying lib
